ocaml/xapi/records.ml

```diff
@@ -217,6 +217,23 @@ let message_record rpc session_id message =
       ]
   }
 
+let network_sriov_record rpc session_id network_sriov =
+  let _ref = ref network_sriov in
+  let empty_record = ToGet (fun () -> Client.Network_sriov.get_record rpc session_id !_ref) in
+  let record = ref empty_record in
+  let x () = lzy_get record in
+  { setref=(fun r -> _ref := r; record := empty_record );
+    setrefrec=(fun (a,b) -> _ref := a; record := Got b);
+    record=x;
+    getref=(fun () -> !_ref);
+    fields =
+      [
+        make_field ~name:"uuid"                     ~get:(fun () -> (x ()).API.network_sriov_uuid) ();
+        make_field ~name:"physical-pif-uuid"        ~get:(fun () -> get_uuid_from_ref (x ()).API.network_sriov_physical_PIF) ();
+        make_field ~name:"logical-pif-uuid"         ~get:(fun () -> get_uuid_from_ref (x ()).API.network_sriov_logical_PIF) ();
+        make_field ~name:"requires-reboot"          ~get:(fun () -> string_of_bool (x ()).API.network_sriov_requires_reboot) ();
+        make_field ~name:"configuration-mode"       ~get:(fun () -> Record_util.network_sriov_configuration_mode_to_string (x ()).API.network_sriov_configuration_mode) ();
```

Comment:
Let's remove this, and keep it internal, like the field in the datamodel.

Reply:
ok